modules/services/dunst.nix

```
@@ -64,6 +64,18 @@ in {
         description = "Package providing <command>dunst</command>.";
       };
 
+      configFile = mkOption {
+        type = types.str;
+        default = "${config.xdg.configHome}/dunst/dunstrc";
+        defaultText = "$XDG_CONFIG_HOME/dunst/dunstrc";
+        description = ''
+          Path to configuration file read by dunst.
+          </para><para>
+          Note that the configuration generated by Home Manager will be
+          written to ~/.config/dunst/dunstrc regardless.
```

```suggestion
          written to $XDG_DATA_HOME/dunst/dunstrc regardless.
```

I'd also add a motivating sentence here, something like "This allows using a mutable configuration file generated from the immutable one, useful in scenarios where live reloading is desired."

---

Side note: I copied the “~/.config” part from the description of this module’s `settings` option. Perhaps it should be changed to “$XDG_CONFIG_HOME” there as well?

---

Sure!